koi.gemspec

```diff
@@ -113,6 +113,6 @@ Gem::Specification.new do |s|
 
   # Development Dependencies
   s.add_development_dependency 'karo'
-  s.add_development_dependency 'byebug'
   s.add_development_dependency 'pry-rails'
+  s.add_development_dependency 'engineyard'
```

Comment:
We currently run `ey init` in the bootstrap process so it makes sense to have the engineyard gem as a dependency. 

At the moment, bootstrapping is actually broken when you use it with a new version of Ruby—most of the times we've bootstrapped a koi project, we've already have the EY gem installed, because we're using an existing version of ruby. 

If we wanted to support different hosting providers out of the box, we might want this to be a gem that's added in the bootstrap process based on some `--hosting-provider` flag or something, rather than an engine dev dependency.

Reply:
My preference would be to go the other way, i.e. remove `ey init` from the app generator and have instructions/generator for including that separately.

Reply:
But as a temporary solution this also works.

Reply:
Actually yeah, I agree that ideally koi should be deployment-platform-agnostic. Separate generators for platform-specific config would probably make more sense. We'd need to change the `.karo.yml` that gets created at bootstrap time as well. Sounds like a separate piece of work / pull request, though.